### PR TITLE
MSPSDS-180 Redis & Sidekiq

### PR DESCRIPTION
**Pull request by AnthonyPengelly**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ cache:
 jobs:
   include:
   - stage: test
+    if: branch != MSPSDS-180_redis
```
Comment (broder) [inline .travis.yml]: I'm guessing that we won't need this anymore?

File: .travis.yml [hunk 2]
```diff
@@ -35,13 +36,19 @@ jobs:
       - docker-compose -f docker-compose.yml -f docker-compose.ci.yml down
   - stage: deploy-to-int
     if: branch = master AND type != pull_request
-    script: SPACE=int ./infrastructure/scripts/deploy.sh
+    script: SPACE=int ./infrastructure/scripts/deploy-app.sh
+  - if: branch = master AND type != pull_request
```
Comment (broder) [inline .travis.yml]: Do we need to have this conditional twice?

File: config/redis.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  host: 'redis'
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+production:
+  url: <%= ENV['VCAP_SERVICES'] && JSON.parse(ENV['VCAP_SERVICES'])["redis"][0]["credentials"]["uri"] %>
```
Comment (broder) [inline config/redis.yml]: Did you think about using https://github.com/cloudfoundry/cf-app-utils-ruby?

File: docker-compose.yml [hunk 4]
```diff
@@ -11,22 +11,43 @@ services:
       - "ES_JAVA_OPTS=-Xms750m -Xmx750m"
     volumes:
       - elasticsearch-volume:/usr/share/elasticsearch/data
+  redis:
+    image: redis
+    command: ["redis-server", "--appendonly", "yes"]
```
Comment (broder) [inline docker-compose.yml]: Does `appendonly` mean that the task list never gets cleared?

File: .travis.yml (hunk 2 above)
Comment (AnthonyPengelly) [reply .travis.yml]: I believe so. They are actually different stages (distinguished by the "-" on this line). When a stage isn't given a name, it takes the name of the previous one - i.e. deploy-to-int

**Pull request merged**